### DISCUSSION TITLE
where to look to modify hinting?

### DISCUSSION THREAD

**Discussion by al-caveman** [qutebrowser/qutebrowser#5711] [Q&A] [answered]
following [this](https://github.com/qutebrowser/qutebrowser/issues/1305), can someone please hint me where to look? i'm abnormally dumb, so any extra help is also appreciated.

(no guarantees that i'm doing this. if anyone else wants to do it, please do it and don't wait for me. i hope no one is going to wait for me)

**Comment by toofar** [comment, marked as answer]
I haven't followed the hinting code very closely but probably have to introduce the context of a "selected" label in HintContext. Add a command to cycle selection. Bind that to tab and handle having a selected hint in follow_hint(). Handle the selected hint not being visible anymore when cycling hint selection and following (or just clear selection on any further filtering like ninewise selected).
